styleguide/voice-tone.md

```diff
@@ -80,8 +80,10 @@ need to be defined if you use it:
 >The Close() method is idempotent, meaning that you can call it multiple times and the effect is
 the same as if you called it once.
 
-### Avoid split infinitives
-While to boldly go may sound good on TV, split infinitives can reduce the readability of your content. So, to go boldly...
+### Use split infinitives correctly
+From the Chicago Manual of Style:
+
+> Sometimes it is perfectly appropriate to split an infinitive with an adverb to add emphasis, clarify meaning, or produce a natural sound. ... A verb’s infinitive or *to* form is split when an intervening word immediately follows to {to bravely assert}. If the adverb bears the emphasis in a phrase {to boldly go} {to strongly favor}, the split infinitive is justified and often even necessary. But if moving the adverb to the end of the phrase doesn’t suggest a different meaning or impair the sound, then you have an acceptable way to avoid splitting the infinitive. Recasting a sentence just to eliminate a split infinitive or to avoid splitting the infinitive can alter the nuance or meaning of the sentence. For example, *it’s best to always get up early* (always modifies get up) is not quite the same as *it’s always best to get up early* (always modifies best). It can also make the phrasing sound unnatural—e.g., *it’s best to get up early always*.
```

Comment:
it's kind of unfortunate the citation uses something we don't recommend (e.g.) 😄